stubs/inertia/tailwind.config.js

```diff
@@ -1,4 +1,6 @@
-const defaultTheme = require('tailwindcss/defaultTheme');
+import defaultTheme from 'tailwindcss/defaultTheme';
+import forms from '@tailwindcss/forms';
+import typography from '@tailwindcss/typography';
```

Comment:
📌 `'tailwindcss/defaultTheme'` does not require, or work with, a prefixed `@` here.